### ISSUE TITLE
Hide Chrome tabs/windows, only showing them in Tab Organizer.

### ISSUE THREAD

**Issue by Pauan** [Pauan/tab-organizer#100]
Originally reported on Google Code with ID 98

```
anonymous:
Suggestion: It would be amazing if there was an option to hide tabs / entire windows
but still keep their information in the organizer. For example, I would love to have
a window with a ton of youtube videos that I may refer to often, but I don't have an
interest in constantly having to see them when I'm not using them.

This becomes annoying if you have multiple windows open in the background and every
time you alt tab or click on the Chrome icon in the task bar you have to select which
window you want to open. If I could have an option to make a specific window invisible
until I call it then I would love you forever.
```

Reported by `pcxunlimited` on 2010-11-12 23:58:59

**Comment by Pauan** [comment]
```
Unfortunately, the only way I can think of doing this is with an NPAPI plugin, and I'm
not sure if that would even work.
```

Reported by `pcxunlimited` on 2010-11-13 00:01:01

**Comment by Pauan** [comment]
```
I would just like to point out that some window managers already support this functionality.

I'm using GNOME + Metacity, and I currently have 2 Workspaces. The 1st Workspace is
for almost everything, and the 2nd Workspace is for Chrome windows only.

When I use Tab Organizer to select a tab, it automatically brings the window from the
2nd Workspace into the 1st Workspace, which is what I want.

I can then manually move Chrome windows into the 2nd Workspace, so they no longer appear
in the taskbar or in the Alt+Tab menu.

It's unfortunate if Windows or Mac OS X don't support this, but it's nice to know that
at least one window manager for GNU/Linux does this.
```

Reported by `pcxunlimited` on 2010-12-02 02:16:14

**Comment by Pauan** [comment]
```
I would also like to point out that Mac OS X Leopard has this functionality, and you
can install a Windows program to enable it on Windows:

http://en.wikipedia.org/wiki/Spaces_(software)
http://dexpot.de/index.php

Thus, this appears to be something that should be provided by the window manager, and
not by Tab Organizer.

However, being able to do this automatically and transparently would be convenient,
so I'll leave this bug open, but at a low priority.
```

Reported by `pcxunlimited` on 2010-12-02 02:26:06
- **Labels added**: Priority-Low
- **Labels removed**: Priority-Medium

**Comment by Pauan** [comment]
```
Seems to be blocked on this Chromium bug...
```

Reported by `pcxunlimited` on 2011-06-18 00:03:37
- **Blocked on**: #78344

**Comment by Pauan** [comment]
Reported by `pcxunlimited` on 2011-06-18 00:18:14
- **Labels added**: Priority-Medium
- **Labels removed**: Priority-Low

**Comment by Pauan** [comment]
```
This is now possible due to the virtual tab/group system that will be used in version
5.
```

Reported by `pcxunlimited` on 2012-08-27 22:08:24
- **Status changed**: `Delegated`
- **Labels added**: Milestone-5

**Comment by Pauan** [comment]
Reported by `pcxunlimited` on 2012-08-27 22:09:17
- **No longer blocked on**: #78344

**Comment by Pauan** [comment]
Reported by `pcxunlimited` on 2012-08-27 23:04:23
- **Labels added**: Type-External
- **Labels removed**: Type-Feature

**Comment by Pauan** [comment]
Reported by `pcxunlimited` on 2012-08-27 23:32:45
- **Labels added**: Type-Visible
- **Labels removed**: Type-External

**Comment by Pauan** [comment]
Reported by `pcxunlimited` on 2012-08-27 23:36:10
- **Labels added**: Type-Confirmed
- **Labels removed**: Type-Visible

**Comment by Pauan** [comment]
Reported by `pcxunlimited` on 2012-08-27 23:39:18
- **Labels removed**: Priority-Medium

**Comment by Pauan** [comment]
Reported by `pcxunlimited` on 2012-08-27 23:47:28
- **Status changed**: `Assigned`

**Comment by Pauan** [comment]
```
Issue 165 has been merged into this issue.
```

Reported by `pcxunlimited` on 2013-02-01 22:11:35
